UX-STANDARDS.md

```diff
@@ -0,0 +1,11 @@
+# UX Standards
+
+These standards include some basic user experience guidelines we align to on the marketing website, docs, and the registry.
+
+- Every page should have exactly one primary action.
+- Use descriptive text for button labels and links, and avoid labels such as "Learn more".
+- Use unique labels for distrinct actions.
+- Use existing components over creating new components unless you identify a need that is not addressed by the current components.
+- Use line lengths between 45 – 75 characters.
```

Comment:
For Markdown, I do not think this is desirable. Because whitespace is meaningful (especially when suggesting edits), I would recommend Markdown paragraphs go on a single line and instead use soft wrap in the editor.

Reply:
I will update this, this is specific to the user experience not the markdown that creates that experience.

Reply:
So we're basically referring to the CSS that would set the width of the enclosing element?

Reply:
Unfortunately, our css is kind of a mess so there's not a single rule that will fix this everywhere.

Reply:
I actually just poked into this, and we could probably do a global-ish fix pretty easily with the `ch` unit for width, if you wanted to make a ticket.  It wouldn't fix everything if we have overriding specificity, but could be a good start.

Reply:
cool, will do!